contracts/colony/ColonyExpenditure.sol

```diff
@@ -230,6 +220,32 @@ contract ColonyExpenditure is ColonyStorage {
     }
   }
 
+  function setExpenditureValues(
+    uint256 _id,
+    uint256[] memory _recipientSlots,
+    address payable[] memory _recipients,
+    uint256[] memory _skillIdSlots,
+    uint256[] memory _skillIds,
+    uint256[] memory _claimDelaySlots,
+    uint256[] memory _claimDelays,
+    uint256[] memory _payoutModifierSlots,
+    int256[] memory _payoutModifiers,
+    address[] memory _payoutTokens,
+    uint256[][] memory _payoutSlots,
+    uint256[][] memory _payoutValues
+  )
+    public
+    stoppable
+    expenditureDraft(_id)
+    expenditureOwnerOrSelf(_id)
```

Comment:
Why does the modifier have to become `expenditureOwnerOrSelf` for functions other than `setExpenditurePayouts`?

Reply:
I didn't want to have two modifiers, mostly. But if you feel strongly I can re-introduce it.

Reply:
Given calls-to-self are how metatransactions work, I'd like to guard against those going wrong wherever possible.